### PR TITLE
fix default name and fs mkdir recursive

### PR DESCRIPTION
**Pull request by jeanawhou**
Ticket link: https://app.ora.pm/p/327217?c=5381452
Fix for double `.png` on filename specially on mac

https://user-images.githubusercontent.com/19387007/117023720-ef9a5500-ad2b-11eb-940a-0d359ef5d9bb.mp4

### CODE REVIEW

File: main/modules/helpers.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ function takeScreenshot() {
     } else {
       const filters = [{ name: 'PNG file', extensions: ['png'] }]
       const fileName = dialog.showSaveDialogSync(win, { filters })
-      if (fileName) fs.writeFile(fileName + '.png', image.toPNG(), () => {})
```
Comment (Quiescent) [inline main/modules/helpers.js:36]: Are you sure about this change?  I tested this by changing the `if` to always pick this branch and found that it needs the '.png' extension to be added.

**Review by Quiescent — CHANGES_REQUESTED** [review]
Please check that extension for the prod branch.  I found that it was needed.

File: main/modules/helpers.js (hunk 1 above)
Comment (jeanawhou) [reply main/modules/helpers.js:36]: @Quiescent  I tested this on a MBA big sur and ubuntu focal on `prod` branch and it still worked for me

**Comment by jeanawhou** [comment]
@Quiescent i tested this on a mba bigsur and ubuntu focal from `prod` branch and it works on my end.

https://user-images.githubusercontent.com/19387007/117343801-3e382280-aed7-11eb-955e-1872f2bdb359.mp4

**Comment by Quiescent** [comment]
> @Quiescent i tested this on a mba bigsur and ubuntu focal from `prod` branch and it works on my end.
> 2021-05-07.01-49-36.mp4

Gosh, sorry!  That's not what I meant. XD
I meant for you to check the prod branch *of the if statement*! :)

i.e. when Plottr is in "production mode" it does something different here and I found that your change for production might be faulty.

File: main/modules/helpers.js (hunk 1 above)
Comment (Quiescent) [reply main/modules/helpers.js:36]: I'm talking about this when I refer to the "prod" branch of the if statement.
You can test this logic by setting the condition to `false` in the if statement.

**Comment by jeanawhou** [comment]
@Quiescent tested on both ubuntu and macos not on windows yet

**Comment by cameronsutter** [comment]
Any update on this?

**Comment by Quiescent** [comment]
> Any update on this?

In just a minute, I'm going to boot windows and make sure that it's working on prod there too.

**Comment by Quiescent** [comment]
I checked on Windows, and it looks good.  LGTM!

**Pull request merged**